static/assets/views/entities.html

```diff
@@ -0,0 +1,43 @@
+
+<h1>{{ctrl.title}} Entity Reference</h1>
+
+<p>
+    This topic describes the entity and type representations for common items that the {{ctrl.title}} API uses. To learn
+    about how to set up your developer account and work with the SKY API, see the
+    <a ng-href="{{ ctrl.getStartedUrl }}">Getting Started</a> guide.
+</p>
+
+<p ng-show="ctrl.showErrorMessage" class="alert alert-danger">
+    There was an issue loading the Entity Reference for the {{ctrl.title}} API.
+</p>
+
+<ul id="entities">
+    <li ng-repeat="entity in ctrl.entities" style="margin: 0px">
+        <a ng-href="#{{entity.name}}">{{entity.name}}</a>
+    </li>
+</ul>
+
+<div ng-repeat="entity in ctrl.entities">
+    <a ng-attr-name="{{entity.name}}"></a>
+    <p>&nbsp;</p>
```

Comment:
Tried top margin and it didn't work as well. Tried LOTS of margin and padding things. The blank p just worked better. So ... yolo'd it. :)